.github/workflows/dispatcher.yml

```
@@ -0,0 +1,18 @@
+name: Pull Request Dispatcher
+on:
+  issue_comment:
+    types:
+      - created
+jobs:
+  slash_command_dispatch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Slash Command Dispatch
+        uses: peter-evans/slash-command-dispatch@v2
+        with:
+          token: ${{ secrets.PAT }}
+          commands: |
+            init
```

This command should be "test" as we want to kick off the entire workflow with a single command; also, it must match one of the `repository_dispatch.types` defined in `handler.yml` when combined with the `event-type-suffix`.

```suggestion
            test
```

---

I'm not sure I understand. running `/ test` will kick off everything? or are we going for multiple slash commands?

---

Right, `/test` will kick off everything.